swap_meet/vendor.py

```diff
@@ -1,2 +1,81 @@
+from swap_meet.item import Item
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
```

Comment:
These imports actually aren't needed! It's good practice once you're done coding to go back and check to see if you accidentally imported things you don't need and remove them.